Testsuite: model.product.ProductVaultTest
Tests run: 0, Failures: 0, Errors: 1, Time elapsed: 0.042 sec

Testcase: model.product.ProductVaultTest took 0 sec
	Caused an ERROR
null
java.lang.NullPointerException
	at model.storage.StorageManager.getAppropriateDAO(StorageManager.java:38)
	at model.productcontainer.StorageUnitVault.saveNew(StorageUnitVault.java:108)
	at model.productcontainer.StorageUnit.save(StorageUnit.java:62)
	at model.product.ProductVaultTest.setUpBeforeClass(ProductVaultTest.java:35)

